plugins/services/src/js/components/modals/EditServiceModal.js

```diff
@@ -19,9 +24,18 @@ class EditServiceModal extends Component {
 
     // Service not found
     if (!service) {
-      this.props.history.push("/services/404");
```

Comment:
I believe it's a forgotten leftover of the react router migration. Please look for `hashHistory.push` as an example.

Reply:
It seems that way to me.

Reply:
That would be a sep issue. Does it show the 404 page correctly though?

Reply:
It does.

Reply:
cool! let's adjust the test and merge it!